scripts/integration_test_contracts.sh

```diff
@@ -31,7 +31,7 @@ key=$(echo "${result}" | jq '.logs[] | .events' | jq '.[0]' | jq '.attributes[2]
 value=$(echo "${result}" | jq '.logs[] | .events' | jq '.[0]' | jq '.attributes[2]' | jq '.value')
 expected_key='"module"'
 expected_val='"wasm"'
-check_result "${key}" "${value}" "expected"
+check_result "${key}" "${value}" "${result}"
```

Comment:
How about adding info on which step fails, for example `"store: ${result}"`?

Reply:
fixed here
[chore: add info on which step fails to error message of check_result](https://github.com/Finschia/finschia-wasm/pull/3/commits/0549f9e2f84abb935aaf2c49d92f3884797d68e3)